.github/workflows/tests-nixtla.yaml

```
@@ -1,4 +1,4 @@
-name: StatsForecast tests
+name: nixtla tests
```

```suggestion
name: Nixtla tests
```

just for consistency with other workflows 🙂